shpc/main/modules/base.py

```diff
@@ -100,34 +100,38 @@ def uninstall(self, name, force=False):
 
         # Ask before deleting anything!
         if not force:
-            msg = name + "?"
+            msg = "Do you wish to uninstall " + name + "?"
             if views_with_module:
                 msg += (
                     "\nThis will uninstall the module from views:\n  %s\nAre you sure?"
                     % "\n  ".join(views_with_module)
                 )
             if not utils.confirm_action(msg, force):
-                return
+                return False # If the user does not want to uninstall
```

Comment:
I used it when I had a different logic if the user decided to say no to uninstallation. But it is currently redundant since the is no longer any prompt before uninstallation. So I will remove it